spark/dl/src/main/scala/com/intel/analytics/bigdl/example/dlframes/imageInference/README.md

```diff
@@ -0,0 +1,65 @@
+## Overview
+   Deep Learning Frames provides high-level APIs for scalable deep learning in Scala with Apache Spark.
+   The current version of Deep Learning Frames provides a suite of tools around working with and processing images using deep learning. 
+   Two exmaples demostrates how to use BigDL to apply popular iamge deep learning models at scale.
```

Comment:
the readme can focus on one example only.

Reply:
ok